eth/ethconfig/config.go

```diff
@@ -285,6 +285,10 @@ type Sync struct {
 	BreakAfterStage            string
 	LoopBlockLimit             uint
 
+	SnapshotPrune             bool
```

Comment:
this variable related only for  "block files"? if yes - better add info about it to name. and maybe comment.

Reply:
Removed in favor of --prune="b"